.github/workflows/gradle.yml

```diff
@@ -228,7 +228,18 @@ jobs:
         run: git --no-pager diff && test -z "$(git --no-pager diff)"
 
       - name: Build
-        run: SUB_BUILD=PLATFORM ./gradlew build javadoc --scan
+        run: |
+            SUB_BUILD=PLATFORM ./gradlew build javadoc --scan
+
+      - name: Integration test
```

Comment:
The more standard way of doing this is to have an `if: github.ref <some comparison>`. The problem with the current approach is that if you're just scanning the list of steps in the Github UI, it's hard to tell which of the two options ran. If instead you have two steps (one is run all integration tests, the other is run just fast integration tests), one will show as disabled and it will be extremely clear which one ran and which didn't.

Reply:
Thanks, I will change that on Monday and sync-up wit @lmossman. For the approval.